docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -118,6 +128,17 @@ The minimal age of a query in the history before it is expired. An expired
 query is removed from the query history buffer and no longer available in
 the :doc:`/admin/web-interface`.
 
+``query.remote-task.max-error-duration``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Default value:** ``5m``
+
+The amount of time that a remote task is allowed to execute while failing to
```

Comment:
why remote? Is it really just about remote to the coordinator?

Reply:
I don't know myself, I inferred this from release notes and other places this property was mentioned. Someone else should know more about it.

Reply:
Better wording is needed here. It's more like if coordinator loses updates of a task for more than 5 minutes, then it will treat the task as failed.

Reply:
Is it more correct to say that the coordinator fails to see updates from the task itself, or that it fails to see updates from the worker note that's responsible for executing the task?

Reply:
@arhimondr might have a better idea

Reply:
Updates from the task itself. The hearth beat mechanism is on the task level, even if multiple tasks happen to run on the same worker.

Reply:
That's interesting, thanks for the clarification. Rewrote this description to reflect this info.